packages/typespec-azure-portal-core/src/lib.ts

```diff
@@ -0,0 +1,37 @@
+import { createTypeSpecLibrary } from "@typespec/compiler";
+
+export const $lib = createTypeSpecLibrary({
+  name: "@azure-tools/typespec-azure-portal-core",
+  diagnostics: {
+    fileNotFound: {
```

Comment:
diagnostic messages should be skewer-case by default e.g. `"file-not-found"` instead of `fileNotFound`.

Reply:
fixed :) thank you!